skbeam/core/accumulators/binned_statistic.py

```diff
@@ -620,7 +620,7 @@ def __call__(self, values, statistic=None):
             The values of the selected statistic in each bin.
         """
         # check for what I believe could be a common error
-        if values.shape != self.expected_shape:
+        if tuple(values.shape) != tuple(self.expected_shape):
```

Comment:
Ah, so the right place for this update would probably be in the `__init__`. That seems like a reasonable patch